samples/functions/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@google-cloud/bigtable-samples",
```

Comment:
I would pull these dependencies up to `samples/package.json`, since the function framework is used in testing, but not part of the sample itself (no need to have a different `package.json`).